README.md

```diff
@@ -117,18 +117,19 @@ The focus of this milestone will be to support synchronization to other
 instances to replicate the state. A synchronization library should also
 be developed for this milestone.
 
-- [ ] :runner: Add support for Ava Labs generic test tool via grpc client
+- [x] Migrate to a fully async interface
```

Comment:
This definitely isn't done. Sure, the interface uses async signatures, but none of the actual execution is async.

Reply:
After a discussion offline, I've changed my mind here.